route_following_plugin/include/route_following_plugin_config.hpp

```diff
@@ -0,0 +1,58 @@
+#pragma once
+/*
+ * Copyright (C) 2022 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#include <string>
+
+namespace route_following_plugin
+{
+    /**
+   * \brief Stuct containing config values values for route_following_plugin
+   */
+   
+  struct Config
+  {
+    // Minimal duration of maneuver, loaded from config file
+    double min_plan_duration_ = 16.0;
+    //Tactical plugin being used for planning lane change
+    std::string lane_change_plugin_ = "CooperativeLaneChangePlugin";
```

Comment:
all the plugin names changed recently to snake case such as "cooperative_lanechange"

I believe the change will show up if you merge develop into your branch, but also make sure that these names are in snake_case in config/parameters.yaml as well

Reply:
notice the name doesn't have "_plugin" in this one.